### PR TITLE
[PrivateLink] Update StorageSync SDK with 2020-03-01

### PR DESCRIPTION
**Pull request by ankushbindlish2**
## Checklist

- [ ] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [ ] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

File: src/StorageSync/StorageSync/Az.StorageSync.psd1 [hunk 1]
```diff
@@ -51,7 +51,7 @@ DotNetFrameworkVersion = '4.7.2'
 # ProcessorArchitecture = ''
 
 # Modules that must be imported into the global environment prior to importing this module
-RequiredModules = @(@{ModuleName = 'Az.Accounts'; ModuleVersion = '1.9.0'; })
+RequiredModules = @(@{ModuleName = 'Az.Accounts'; ModuleVersion = '1.8.1'; })
```
Comment (VeryEarly) [inline src/StorageSync/StorageSync/Az.StorageSync.psd1:54]: any reason this need to be downgrade?

**Comment by VeryEarly** [comment]
Hi @ankushbindlish2 ，

It looks like this PR contains changes of another PR #12240 , should the other it be closed?

File: src/StorageSync/StorageSync/Az.StorageSync.psd1 (hunk 1 above)
Comment (ankushbindlish2) [reply src/StorageSync/StorageSync/Az.StorageSync.psd1:54]: no reason. will revert

**Comment by ankushbindlish2** [comment]
> Hi @ankushbindlish2 ，
> 
> It looks like this PR contains changes of another PR #12240 , should the other it be closed?

closed that PR

File: src/StorageSync/StorageSync/StorageSyncService/SetStorageSyncServiceCommand.cs [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+                    Tags = TagsConversionHelper.CreateTagDictionary(Tag ?? new Hashtable(), validate: true),
+                    IncomingTrafficPolicy = incomingTrafficPolicy
+                };
+
+                Target = string.Join("/", resourceGroupName, resourceName);
+                if (ShouldProcess(Target, ActionMessage))
+                {
+                    StorageSyncModels.StorageSyncService storageSyncService = StorageSyncClientWrapper.StorageSyncManagementClient.StorageSyncServices.Update(ResourceGroupName, Name, updateParameters);
+
+                    WriteObject(storageSyncService);
```
Comment (VeryEarly) [inline src/StorageSync/StorageSync/StorageSyncService/SetStorageSyncServiceCommand.cs:196]: should this be transformed to PSStorageSyncService

Comment (ankushbindlish2) [reply src/StorageSync/StorageSync/StorageSyncService/SetStorageSyncServiceCommand.cs:196]: its a custom function

**Pull request merged**